### PR TITLE
[fix](delete-sign) Disable deletion with delete sign when doing cumulative compaction

### PR DESCRIPTION
**Pull request by Hastyshell**
## Proposed changes

Delete sign could not be applied when delete on cumu compaction is enabled, bucause it is meant for delete with predicates. If delete design is applied on cumu compaction, it will lose effect when doing base compaction.

So disable delete sign when doing cumulative compaction.

### CODE REVIEW

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run p0

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run p1

**Comment by TangSiyang2001** [comment]
run buildall

File: be/src/vec/olap/block_reader.cpp [hunk 1]
```diff
@@ -377,8 +377,7 @@ Status BlockReader::_unique_key_next_block(Block* block, bool* eof) {
         }
     } while (target_block_row < _reader_context.batch_size);
 
-    // do filter delete row in base compaction, only base compaction need to do the job
-    if (_filter_delete) {
+    if (_filter_delete && !_cumu_delete) {
```
Comment (luwei16) [inline be/src/vec/olap/block_reader.cpp:380]: Replace _filter_delete && !_cumu_delete with a new variable

Comment (TangSiyang2001) [reply be/src/vec/olap/block_reader.cpp]: updated

**Review by luwei16 — APPROVED** [review]

**Comment by TangSiyang2001** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**